packages/gateway/src/features/user/schema.graphql

```diff
@@ -55,6 +55,56 @@ type Avatar {
   data: String!
 }
 
+type Search {
+  searchId: String!
+  name: String!
+  event: String
+  registrationStatuses: [String]
+  dateOfEvent: String
+  dateOfEventStart: String
+  dateOfEventEnd: String
+  registrationNumber: String
+  trackingId: String
+  dateOfRegistration: String
+  dateOfRegistrationStart: String
+  dateOfRegistrationEnd: String
+  declarationLocationId: String
+  declarationJurisdictionId: String
+  eventLocationId: String
+  eventLocationLevel1: String
+  eventLocationLevel2: String
+  eventLocationLevel3: String
+  eventLocationLevel4: String
+  eventLocationLevel5: String
+  childFirstNames: String
+  childLastName: String
+  childDoB: String
+  childDoBStart: String
+  childDoBEnd: String
+  childGender: String
+  deceasedFirstNames: String
+  deceasedFamilyName: String
+  deceasedGender: String
+  deceasedDoB: String
+  deceasedDoBStart: String
+  deceasedDoBEnd: String
+  motherFirstNames: String
+  motherFamilyName: String
+  motherDoB: String
+  motherDoBStart: String
+  motherDoBEnd: String
+  fatherFirstNames: String
+  fatherFamilyName: String
+  fatherDoB: String
+  fatherDoBStart: String
+  fatherDoBEnd: String
+  informantFirstNames: String
+  informantFamilyName: String
+  informantDoB: String
+  informantDoBStart: String
+  informantDoBEnd: String
+}
```

Comment:
How about we extract the search parameters into a separate type called `AdvancedSearchParameters` and modify this `Search` schema like this:
```gql
type BookmarkedSearch {
  searchId: String!
  name: String!
  parameters: AdvancedSearchParameters!
}
```
I'm suggesting this because we will be reusing this `AdvancedSearchParameters` in a quite a few places so creating a separate type for this makes more sense. For example we can create a separate Input type for this `AdvancedSearchParametersInput` and then we can reuse this types in the bookmark flow:
```gql
input BookmarkSearchInput {
  name: String!
  userId: String!
  parameters: AdvancedSearchParametersInput!
```